lib/fileutils.rb

```diff
@@ -1274,8 +1274,17 @@ def door?
     def entries
       opts = {}
       opts[:encoding] = ::Encoding::UTF_8 if fu_windows?
-      Dir.children(path, opts)\
-          .map {|n| Entry_.new(prefix(), join(rel(), n.untaint)) }
+
+      files = begin
```

Comment:
I prefer to assign from result of `if` statement directly instead of `begin` ... `end` statement.